query1.php

```diff
@@ -59,39 +60,43 @@
 								{	
 								?>
 								<!--php is above. HTML is below. Used to output the query results-->
-								<p><?php echo $output['Total_Time']; ?></p>
+								<center><p><?php echo $output['Total_Time']; ?></p></center>
 				
 								<?php
 								//closes the output while loop
 								}
 								?>
-			</div>
+			</div></center>
 
 			<!-- This class is for my main content-->	
 			<div class="content"> 
 				
                         <heading1>
-                                <Song_ID1 style="padding-left: 5px;"><h4>#</h4></Song_ID1>
-                                <Title1><h4>Title</h4></Title1>
-							    <Artist1><h4>Artist</h4></Artist1>
-                                <Album1><h4>Album</h4></Album1>
-								<Genre1><h4>Genre</h4></Genre1>
-								<Duration1><h4>Duration</h4></Duration1>
-                                <Size1><h4>Size</h4></Size1>
+                                <Song_ID1 style="padding-left: 5px;"><h2>#</h2></Song_ID1>
+                                <Title1><h2>Title</h2></Title1>
+							    <Artist1><h2>Artist</h2></Artist1>
+                                <Album1><h2>Album</h2></Album1>
+								<Genre1><h2>Genre</h2></Genre1>
+								<Duration1><h2>Duration</h2></Duration1>
+                                <Size1><h2>Size</h2></Size1>
                         </heading1>            
 				
                         	<?php
 								require "3.3_Assessment_FTuason_mysqli.php";
 					
 								//creates a variable to store the sql query that displayed all song info sorted by title then artist (both highest first)
-								$query = ("SELECT s.Song_ID, s.Title, s.Duration, s.Size, a.Album, r.Artist, g.Genre
+								$query = ("SELECT s.Song_ID, s.Title, s.Duration, s.Size,
```

Comment:
Updated my query to show the iterative improvements I made in my php database (concatenation)